### PR TITLE
Add NNP documentation page - WIP

### PR DESCRIPTION
**Pull request by cschran**

### CODE REVIEW

**Comment by oschuett** [comment]
Thanks a lot for contributing documentation! To fix the precommit error simply run `make pretty`.

File: docs/methods/machine_learning/nnp.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 # Neural Network Potentials
 
-Unfortunately no one has gotten around to writing this page yet :-(
+## Input Section
+
+NNPs in CP2K are supported via the `NNP` Force_Eval section. As an example, a basic `NNP` section would be:
```
Comment (oschuett) [inline docs/methods/machine_learning/nnp.md:5]: One can link directly to input sections like this: `[NNP](#CP2K_INPUT.FORCE_EVAL.NNP)` and to individual keywords like this: `[NNP_INPUT_FILE_NAME](#CP2K_INPUT.FORCE_EVAL.NNP.NNP_INPUT_FILE_NAME)`.

Comment (cschran) [reply docs/methods/machine_learning/nnp.md:5]: perfect, thanks! I'll prepare a more substantial documentation in the next days.

**Comment by oschuett** [comment]
Gentle ping :-)

**Comment by oschuett** [comment]
How shall we proceed with this PR? Should I just merge it as is?

**Comment by cschran** [comment]
Sorry, very busy summer (and now fall)! I'll close this for now until I actually make a proper effort with this.

**Pull request closed without merging**